metashare/api/serializers.py

```diff
@@ -83,6 +83,38 @@ def get_repo_url(self, obj) -> Optional[str]:
         return f"https://github.com/{obj.repo_owner}/{obj.repo_name}"
 
 
+class AuthorCommitSerializer(serializers.Serializer):
+    name = serializers.CharField(required=False)
+    email = serializers.CharField(required=False)
+
+
+class CommitSerializer(serializers.Serializer):
+    id = serializers.CharField()
+    timestamp = serializers.CharField()
```

Comment:
I don't see `commit.timestamp` documented anywhere in the GH webhook API: https://developer.github.com/v3/activity/events/types/#pushevent